src/components/post/PostRelated.vue

```diff
@@ -33,11 +31,15 @@ export default {
   computed: {
     posts: {
       get: function () {
-        let post = this.post
+        const post = this.post
         const store_posts = this.$store.state.posts
-        const posts = Object.keys(store_posts).map(post => store_posts[post])
+        const posts = Object.keys(store_posts)
+          .map(post => store_posts[post])
+          .filter(p => (p.link && p.link.search(/software(engineering|)daily\.com/g) >= 0))
```

Comment:
you probably don't need `>= 0`. If the return is `0`, this is false. Anything else will be true

Reply:
It needs to be true if it's `0` as well. We only want links that have either softwaredaily.com or softwareengineeringdaily.com